benchmark.sh

```diff
@@ -1,35 +1,19 @@
 #!/bin/bash
 
-# Benchmark Drupal site.
+# Benchmark the Drupal site.
 
-set -xeuo pipefail
-
-# Check if the first parameter is either "ddev" or "lando".
+# Ensure the first parameter is either "ddev" or "lando".
 if [[ $# -lt 1 || ( "$1" != "ddev" && "$1" != "lando" ) ]]; then
   echo "Error: First parameter must be either 'ddev' or 'lando'."
   exit 1
 fi
 
-# Get login URL and remove \r fron the end.
+# Get the Drupal login URL with drush.
 login_url="$($1 drush uli)"
-login_url="${login_url//[$'\r']}"
-
-# Log in and output cookies to stdout.
-drupal_session_cookie=$(curl \
-  --insecure \
-  --max-redirs 5 \
-  --cookie-jar - \
-  "$login_url" | grep -Eo 'SSESS[a-z0-9]+\s[a-zA-Z0-9%-]+')
-
-# Replace space with = to make it acceptable parameter for ab tool.
-drupal_session_cookie=$(sed 's/\s/=/g' <<< "$drupal_session_cookie")
-
-# In some OS's \s does not match tabs so let's try and catch that here.
-drupal_session_cookie=$(sed 's/\t/=/g' <<< "$drupal_session_cookie")
 
-# Run ab tests.
+# Run load tests with Locust.
 if [[ "$1" == "ddev" ]]; then
-  ab -C ${drupal_session_cookie} -n 50 -l https://drupal-benchmark.ddev.site/admin/modules
+  ULI="$login_url" locust --headless --users 1 --spawn-rate 1 --run-time 30 --stop-timeout 5 --only-summary -H https://drupal-benchmark.ddev.site
```

Comment:
We can use config files to have less repetition https://docs.locust.io/en/latest/configuration.html